config/initializers/inflections.rb

```diff
@@ -3,15 +3,15 @@
 # Add new inflection rules using the following format. Inflections
 # are locale specific, and you may define rules for as many different
 # locales as you wish. All of these examples are active by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
+# ActiveSupport::Inflector.inflections(:eng) do |inflect|
 #   inflect.plural /^(ox)$/i, '\1en'
 #   inflect.singular /^(ox)en/i, '\1'
 #   inflect.irregular 'person', 'people'
 #   inflect.uncountable %w( fish sheep )
 # end
 
 # These inflection rules are supported but not enabled by default:
-ActiveSupport::Inflector.inflections(:en) do |inflect|
+ActiveSupport::Inflector.inflections(:eng) do |inflect|
```

Comment:
Revert this change please. This symbol has nothing to do with Sara Alert languages ([see this](https://api.rubyonrails.org/classes/ActiveSupport/Inflector/Inflections.html)), and so this breaks things.

Reply:
sounds good, reverting now

Reply:
The comment above needs the reversion as well. Also, this presumably was the result of a find and replace all, so please take a min to go back and inspect everywhere we have `:eng` to make sure there are no other unintended changes.

Reply:
Fixed! It wasn't a find and replace, but I will go through again and double check everything.